### PR TITLE
[16.0] web editor: record info on image conversion to attachments

### PR DESCRIPTION
**Pull request by cammarosano**
[FIX] web_editor: skipped image conversion to attachment in HtmlField
[FIX] web_editor: missing record info for converted images

task-3495668

### CODE REVIEW

**Comment by cammarosano** [comment]
@fw-bot ignore

PR for saas-16.1+: https://github.com/odoo/odoo/pull/134704

File: addons/web_editor/static/tests/html_field_tests.js [hunk 1]
```diff
@@ -379,6 +379,8 @@ QUnit.module("WebEditor.HtmlField", ({ beforeEach }) => {
                 route === `/web_editor/modify_image/${imageRecord.id}`
             ) {
                 if (modifyImageCount === 0) {
+                    assert.ok(args.res_model);
+                    assert.ok(args.res_id);
```
Comment (dmo-odoo) [inline addons/web_editor/static/tests/html_field_tests.js:383]: Wouldn't it be better to test the actual values rather ?

Comment (cammarosano) [reply addons/web_editor/static/tests/html_field_tests.js]: Indeed. Done!

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**